R/bind-cache.R

```diff
@@ -4,26 +4,19 @@ utils::globalVariables(".GenericCallEnv", add = TRUE)
 #'
 #' @description
 #'
-#' `bindCache()` adds caching to the following kinds of objects used in Shiny:
+#' `bindCache()` adds caching [reactive()] expressions and `render*` functions
+#' (like [renderText()], [renderTable()], ...).
 #'
-#' * [reactive()]` expressions.
-#' * `render*` functions, like [renderText()], [renderTable()], and so on.
+#' Ordinary [reactive()] expressions automatically cache their _most recent_
+#' value, which helps to  avoid redundant computation in downstream reactives.
+#' `bindCache()` will cache all previous values (as long as they fit in the
+#' cache) and they can be shared across user sessions. This allows
+#' `bindCache()` to dramatically improve performance when used correctly.
 #'
-
```

Comment:
These extra blank lines are because of RStudio messing up formatting when I have it reflow the text (https://github.com/rstudio/rstudio/issues/8378). If you tell RStudio reflow the text, it'll only do it for the current block (and won't mess up the other blocks).

Reply:
Should be better now